README.md

```diff
@@ -19,6 +19,23 @@ Framework](https://dotnet.microsoft.com/download/dotnet-framework) except for
 `.NET Framework 3.5 SP1`. Any exceptions to this are noted in the individual
 `README.md` files.
 
+## OpenTelemetry .NET and relation with .NET Activity API
```

Comment:
For folks who don't need Activity API (e.g. developers who just need to configure/enable telemetry) this part is less important. Consider moving this to the tracing part.